akka-remote/src/test/scala/akka/remote/artery/ArterySmokeTest.scala

```diff
@@ -11,7 +11,11 @@ object ArterySmokeTest {
   val commonConfig = """
      akka {
        actor.provider = "akka.remote.RemoteActorRefProvider"
-       remote.artery.enabled = on
+       remote.artery {
+         enabled = on
+         #transport = tcp
+         transport = aeron-udp
```

Comment:
@drewhk when I run this test with `tcp` there is an exception during system shutdown, and it's the same before my changes. Can you take a look at that?
"GraphStageLogic$StageActorRefNotInitializedException: You must first call getStageActorRef, to initialize the Actors behaviour"